bin/cli

```diff
@@ -9,14 +9,15 @@ if (!file_exists($autoloadPath)) {
     exit(1);
 }
 
-set_error_handler(function ($errno, $errstr, $errfile, $errline, array $errcontext): bool {
-    if (!(error_reporting() & $errno)) {
-        // respect error_reporting() level
-        // libraries used in custom components may emit notices that cannot be fixed
-        return false;
+set_error_handler('exceptions_error_handler');
+function exceptions_error_handler($severity, $message, $filename, $lineno)
```

Comment:
Mělo by to mít $errorContext myslím, protože ten error handler to bude volat s 5 parametry. Že se nepoužije nevadí IMO

Reply:
Zbytečně by to mohlo házet nějakou notice `too many parameters` nebo tak něco v dalších verzích PHP...

Reply:
diky doplnim